data/services/aws-rds.toml

```diff
@@ -1,3 +1,5 @@
+# Service metadata lives
+# https://github.com/18F/aws-broker/blob/59f949b/catalog-template.yml#L8
```

Comment:
Actually the best resolution for this might be to skip thinking about it for now, just put in any URL that serves as a helpful hint, and hope for the glorious future of https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-3375 :)